### ISSUE TITLE
Should a GBFS feed be added for Montreal's bike sharing system?

### ISSUE THREAD

**Issue by nekohayo** [public-transport/transitous#74]
Montréal's "Bixi" bikesharing system does offer GTFS data: https://bixi.com/en/open-data/ (at the bottom)

Probably other big cities (Paris, Toronto, Lyon, etc.?) have similar systems, I think.

They're part of the public transit systems in a way, I'm not sure exactly how @mlundblad might integrate this in GNOME Maps from a UI perspective, though!

**Comment by jbruechert** [comment]
I don't think so, now only the metadata needs to be extended to handle it. Should be easy enough if you know some python.

**Comment by felixguendling** [comment]
I think at the moment the routing performance is still not optimal. Especially if there are many vehicle types in one area (because each one has to be routed separately). Also: the API is not stable yet (we plan to make some changes to make it more flexible). But adding feeds to transitous is still a good idea to get a first clue of what other problems might exist. Less testing = more bugs.
